src/pretix/presale/templates/pretixpresale/index.html

```diff
@@ -2,24 +2,59 @@
 {% load bootstrap3 %}
 {% load i18n %}
 {% load static %}
+
 {% block content %}
-    <div class="form-signin" action="" method="post">
-        <h3>{% trans "Hello!" %}</h3>
-        <p>
-            {% blocktrans trimmed with a_attr="href='https://eventyay.com'" %}
-                This is a self-hosted installation of <a {{ a_attr }}>eventyay, your open source event solution.</a>.
-            {% endblocktrans %}
-        </p>
-        <p>
-            {% blocktrans trimmed %}
-                If you're looking to buy a ticket, you need to follow a direct link to an event or organizer profile.
-            {% endblocktrans %}
-        </p>
-        <p>
-            {% blocktrans trimmed with a_attr="href='/control/'" %}
-                If you're looking to configure this installation, please <a {{ a_attr }}>head over here</a>.
-            {% endblocktrans %}
-        </p>
-        <p>{% trans "Enjoy!" %}</p>
+<div class="container">
+    <div class="form-wrapper">
+        <div class="form-signin">
+            <h3>{% trans "Hello!" %}</h3>
+            <h2>{% trans "WELCOME TO EVENTYAY-TICKETS" %}</h2>
+            <p>
+                {% blocktrans trimmed with a_attr="href='https://eventyay.com'" %}
+                    This is a self-hosted installation of <a {{ a_attr }}>eventyay, your open source event solution.</a>.
```

Comment:
It is a bad practice to put code (like `<a>`, `</a>`) in to-be-translated string. Because those strings will be handovered to translators, who are not tech persons and will likely put wrong code, which will break our site when the translated strings are applied.

But this mistake is from the original pretalx code. It is not urgent to fix right now.